packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -28,7 +30,7 @@ addReplyUrlIfNotExists() {
 
     # Get existing reply urls of the app registration
     echo "Fetching existing replyUrls..."
-    replyUrls=$(az ad app show --id $clientId --query "replyUrls" -o tsv)
+    replyUrls=$(az ad app show --id $clientId --query "replyUrls" -o tsv) || true
```

Comment:
why is this change needed?

Reply:
because we still try to add in both cases, ClientId was passed or a new app was created, so we might need to face the same permission issue if we are running from a pipeline